content/tbb/tbb-43/contents.lr

```diff
@@ -0,0 +1,10 @@
+_model: question
+---
+title: What is the difference between using Tor Browser and 'Incognito mode' or private tabs?
+---
+seo_slug: difference-between-tor-browser-incognito-mode
+---
+description:
+While the names may imply otherwise, 'Incognito mode' and 'private tabs' do not actually protect the identity of the user. They erase all the information on your machine relating to the browsing session after they are closed, but have no measures in place to hide your acivity or digital fingerprint online. This means that an observer can collect your traffic just as easily as any regular browser.
+
+Tor Browser offers all the amnesic features of private tabs while also hiding the source IP, browsing habits and details about a device that can be used to fingerprint activity across the web, allowing for a truly private browsing session that's fully obfuscated from end-to-end.
```

Comment:
What about linking Mozilla Firefox "Common Myths about Private Browsing":
https://support.mozilla.org/en-US/kb/common-myths-about-private-browsing

Reply:
All adjustments should be present now :) Let me know if there's anything else you'd like to include.